modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/platform/MobileContextMenu.ts

```diff
@@ -101,28 +102,42 @@ const setupiOSOverrides = (editor: Editor) => {
   };
 };
 
-const show = (editor: Editor, e: EditorEvent<TouchEvent>, items: MenuItems, backstage: UiFactoryBackstage, contextmenu: AlloyComponent, useNodeAnchor: boolean, highlightImmediately: boolean) => {
-  const anchorSpec = useNodeAnchor ? getNodeAnchor(editor) : getPointAnchorSpec(editor, e);
+const getAnchorSpec = (editor: Editor, e: EditorEvent<TouchEvent>, anchorType: ContextMenuAnchorType) => {
```

Comment:
Seeing as this is duplicated, I'm wondering if it might be worth moving this (as well as the `ContextMenuAnchorType` definition) into a single shared file? Possibly even `../Coords.ts`, seeing as that's where the individual get functions are

Reply:
Agreed, I hadn't got around to commenting yet but this was going to be one of the changes I was going to request.

Reply:
Putting it in `Coords` would probably be a reasonable place given that's where the current anchors are generated.

Reply:
Moved to Coords.